definitions/control-v1.yaml

```diff
@@ -3548,6 +3554,253 @@ components:
       - requestMode
       - source
       - target
+    kafka_rule_post:
+      type: object
+      additionalProperties: false
+      properties:
+        status:
+          type: string
+          description: The status of the rule. Rules can be enabled or disabled.
+          example: enabled
+        ruleType:
+          type: string
+          description: The type of rule. In this case Kafka (using Firehose). See the <a href="https://ably.com/documentation/general/firehose">Ably documentation</a> for further information.
+          enum:
+          - kafka
+        requestMode:
+          type: string
+          description: This is Single Request mode or Batch Request mode. Single Request mode sends each event separately to the endpoint specified by the rule. Batch Request mode rolls up multiple events into the same request. You can read more about the difference between single and batched events in the Ably <a href="https://ably.com/documentation/general/events#batching">documentation</a>.
+          example: single
+        source:
+          "$ref": "#/components/schemas/rule_source"
+        target:
+          type: object
+          additionalProperties: false
+          properties:
+            routingKey:
+              type: string
+              description: The Kafka partition key. This is used to determine which partition a message should be routed to, where a topic has been partitioned.
+              example: 'partitionKey'
+            brokers:
+              type: array
+              description: Brokers host Kafka partitions. You can provide a list of brokers, using the format `host`, `host:port` or `ip:port`.
+              example: ['kafka.ci.ably.io:19092', 'kafka.ci.ably.io:19093']
+              items:
+                type: string
+                example: 'kafka.ci.ably.io:19092'
+            auth:
+              type: object
+              description: The Kafka <a href="https://docs.confluent.io/platform/current/kafka/overview-authentication-methods.html">authentication mechanism</a>.
+              additionalProperties: false
+              properties:
+                sasl:
+                  type: object
+                  description: SASL(Simple Authentication Security Layer) / SCRAM (Salted Challenge Response Authentication Mechanism) uses usernames and passwords stored in ZooKeeper. Credentials are created during installation. See documentation on <a href="https://docs.confluent.io/platform/current/kafka/authentication_sasl/authentication_sasl_scram.html#kafka-sasl-auth-scram">configuring SCRAM</a>.
+                  additionalProperties: false
+                  properties:
+                    mechanism:
+                      type: string
+                      description: The hash type to use. SCRAM supports either SHA-256 or SHA-512 hash functions.
+                      example: 'scram-sha-256'
+                      enum:
```

Comment:
This doesn't render as expected when running locally (no dropdown). I can't get it to work, so I wonder if it's a Redoc bug?